dynamiqs/utils/utils.py

```diff
@@ -550,30 +551,81 @@ def todm(x: Tensor) -> Tensor:
         )
 
 
-def ket_overlap(x: Tensor, y: Tensor) -> Tensor:
-    r"""Returns the overlap (inner product) $\braket{\varphi,\psi}$ between two kets
-    $\ket\varphi$ and $\ket\psi$.
+def overlap(x: Tensor, y: Tensor) -> Tensor:
+    r"""Returns the overlap between two quantum states.
```

Comment:
For the code I don't really care it's longer. For the doc I agree, but I think it's kinda nice to have. Maybe we can keep the feature but document it less?

Reply:
So right now it's only specified in the argument list

Reply:
I think we should just remove support for bras. It will be extremly rarely used, and if necessary one can just do a `toket()` before using the method... But if we choose to keep it we have to document it.